Pds4File/pds4file.py

```diff
@@ -3492,18 +3497,18 @@ def from_abspath(abspath, fix_case=False, must_exist=False,
         # Search for "holdings"
         parts_lc = [p.lower() for p in parts]
         try:
-            holdings_index = parts_lc.index('holdings')
+            #holdings_index = parts_lc.index('holdings')
+            pds4_holdings_index = parts_lc.index('pds4-holdings') # Change variable name to distinguish from PDS3
```

Comment:
One nitpick, 'pds4-holdings' string has been used in multiple places, let's create a global variable, something like ```PDS4_HOLDINGS``` or a better name, to store 'pds4-holdings', and use string interpolation with the global variable when 'pds4-holdings' is used. So in the future if we happen to change the directory name to something else, we can just update the global variable instead of changing 'pds4-holdings' everywhere.